index.d.ts

```diff
@@ -1146,6 +1146,7 @@ declare namespace Dysnomia {
   interface Widget {
     channel_id?: string;
     enabled: boolean;
+    reason?: string;
```

Comment:
This probably isn't the end of the world, but it might be more worth it to split the reason for the edit into a separate interface.